components/omega/configs/Default.yml

```diff
@@ -36,6 +36,11 @@ Omega:
   Tracers:
     Base: [Temperature, Salinity]
     Debug: [Debug1, Debug2, Debug3]
+  Eos:
+    EosType: Linear
+    LinearDRhoDT: -0.2
```

Comment:
Are we likely to ever change the values for linear EOS? Or should they be internal?   If we keep them as config variables, consider using a Linear subgroup so that if we are using TEOS (most of the time), we don't need to include these values in the config. So something like:
``` YAML
Eos:
   EosType: Linear
   Linear:
      DRhoDT: -0.2
```
etc